documentation/plain_rules.md

```diff
@@ -259,6 +261,28 @@ or use the `json` parameter
         }
     }
 ```
+You can also send form encoded data by using the `form` parameter
+
+```json
+ "action": {
+        "type": "post",
+        "parameters": {
+            "url": "http://${target_host}:${target_port}/myapp/${id}",
+            "headers": {
+                        "Content-type": "application/json",
+                        "X-${type}-pressure": "${BloodPressure}"
+            },
+            "qs": {
+                        "${id}": "${BloodPressure}"
+            },
+            "form": {
```

Comment:
Looking to the examples, it seems that "form" and "json" are mutually exclusive. Is my interpretation correct? What would happen if the action includes *both*  "form" and "json" (I don't see anything in the code modifications checking that posibility)?

Reply:
Hi @fgalan @cblanco . I'm looking forward for your comments on this :)

Reply:
Lest's consider the end-to-end perspective in the CEP API. I mean, what should happen in the case the user tries to use both "json" and "form" when creating or updating a rule is that he/she gets a 400 Bad Request response with an error telling about she/he cannot do that. 

In addition:

* An unit test covering this situation should be added to ensure it is working.
* Documentation should explain that both parameters are mutually exclusive.